internal/services/cognitive/cognitive_account_rai_policy_resource_test.go

```diff
@@ -0,0 +1,155 @@
+package cognitive_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/cognitive/2024-10-01/raipolicies"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type RaiPolicyTestResource struct{}
+
+func TestCognitiveAccountRaiPolicy_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_cognitive_account_rai_policy", "test")
+	r := RaiPolicyTestResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		data.ApplyStep(r.basicConfig, r),
+		data.ImportStep(),
+	})
```

Comment:
This looks like a legacy pattern copied from other cognitive resources which are quite old. Please update all the tests to follow the more familiar testing pattern like in this example https://github.com/hashicorp/terraform-provider-azurerm/blob/66c60863f273a31ca6f36ed2ecf57b71fccb78b8/internal/services/devcenter/dev_center_project_environment_type_resource_test.go#L21-L34

Reply:
Will use the suggested testing pattern

Reply:
I follow this in the guidance https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-new-resource.md. I thought this was the newer pattern. But I will use the suggested testing pattern